tests/console/journalctl.pm

```diff
@@ -87,8 +87,10 @@ sub reboot {
         my $instance = publiccloud::instances::get_instance();
         $instance->softreboot();
     } else {
+        # svirt backend needs much longer for reboot
+        my $timeout = (is_svirt_except_s390x && is_sle) ? 600 : 300;
```

Comment:
Why `is_sle`?

Reply:
becasue at moment we have only issue for sle, don't want extend timeout for opensuse

Reply:
That is not valid reason. It may fail also in openSUSE in the future. Just increase the timeout globally.